enterprise-ws-spec/src/main/asciidoc/DeploymentDescriptors.adoc

```diff
@@ -88,63 +88,56 @@ Imported files may import other files as well using relative locations
 or explicit URLs. It is recommended that the WSDL file and relative
 referenced files be packaged in the wsdl directory as described in
 section link:#anchor-52[5.4.1]. Relative references must not start with
-a “/”.
+a "/".
 * **Service QName. **In addition to specifying the WSDL document, the
 developer may also specify the WSDL Service QName in the wsdl-service
-element for each Port defined in the deployment descriptor when JAX-WS
-based runtime is used. This element is required if the port-component
-being defined is a Provider Interface defined by JAX-WS.
-* *Port’s QName.* In addition to specifying the WSDL document, the
+element for each Port defined in the deployment descriptor when Jakarta 
+XML Web Services based runtime is used. This element is required if the 
+port-component being defined is a Provider Interface defined by Jakarta 
+XML Web Services.
+* *Port's QName.* In addition to specifying the WSDL document, the
 developer must also specify the WSDL port QName in the wsdl-port element
 for each Port defined in the deployment descriptor.
 * **MTOM/XOP support. **The developer may specify if MTOM/XOP support
 for the port-component is enabled or disabled by using enable-mtom
-element when JAX-WS based runtime is used.
+element when Jakarta XML Web Services based runtime is used.
 * **Addressing support. **The developer may specify an addressing
-support for the port-component by using addressing element when JAX-WS
-based runtime is used.
+support for the port-component by using addressing element when 
+Jakarta XML Web Services based runtime is used.
 * **RespectBindingg support. **The developer may specify a respect
```

Comment:
typo: `RespectBindingg` -> `Respect Binding`

Reply:
Will fix the typo.